### DISCUSSION TITLE
Integrate alerts.home-assistant.io into Home Assistant

### DISCUSSION THREAD

**Discussion by balloob** [home-assistant/frontend#11186] [Other feature requests]
As a user I would like to be able to see alerts from https://alerts.home-assistant.io/alerts.json related to my installation in the frontend.

- I should be able to be notified if new alerts come available.
- I should by default only see the alerts relevant to my installation
- If a new alert impacts my installation, I should receive a notification to go to the alerts page

**Comment by bramkragten** [comment]
Shouldn’t this be done on the backend, so a user can also use automations and stuff. Maybe a user wants a push message on their phone or...

**Comment by balloob** [comment]
Good point
